tf_encrypted/convert/register.py

```diff
@@ -198,6 +199,33 @@ def _keras_dense(converter, interiors, inputs):
   return out
 
 
+def _keras_batchnorm(converter, interiors, inputs):
+  x_in = converter.outputs[inputs[0]]
+
+  bn_op = interiors["FusedBatchNorm"]
+  fmt = bn_op.attr["data_format"].s.decode('ascii')
+
+  gamma = _nodef_to_numpy_array(interiors["gamma"])
+  beta = _nodef_to_numpy_array(interiors["beta"])
+  moving_mean = _nodef_to_numpy_array(interiors["moving_mean"])
+  moving_variance = _nodef_to_numpy_array(interiors["moving_variance"])
+
+  input_shape = x_in.shape.as_list()
+
+  layer = Batchnorm(input_shape=input_shape,
```

Comment:
can we use the `tfe.keras.layers.BatchNormalization` layer here instead?  we're slowly working on phasing out the tfe.layers module